app/views/shared/_footer.html.erb

```diff
@@ -29,8 +29,8 @@
         <a id="feedback-modal" class="feedback-survey-popup desktop:display-inline-block desktop:grid-col-auto desktop:padding-0 desktop:border-top-0 border-top-1px border-base-light desktop:margin-right-5 grid-col-12 padding-y-2 padding-x-2 text-bold dm-footer-link" href="#">
           Send feedback
         </a>
-        <a class="desktop:display-inline-block desktop:grid-col-auto desktop:padding-0 desktop:border-top-0 border-top-1px border-base-light desktop:margin-right-5 grid-col-12 padding-y-2 padding-x-2 text-bold dm-footer-link" href="<%= nominate_a_practice_path %>" >
-          Nominate a practice
+        <a href="https://www.va.gov/privacy-policy" target="_blank" class="desktop:display-inline-block desktop:grid-col-auto desktop:padding-0 desktop:border-top-0 border-top-1px border-base-light desktop:margin-right-5 grid-col-12 padding-y-2 padding-x-2 text-bold dm-footer-link" aria-controls="dm-terms-and-conditions-modal">
```

Comment:
Can we remove the `aria-controls='dm-terms-and-conditions-modal'`?

Reply:
Good catch Josh - I removed it.